evm/src/NttManager/ManagerBase.sol

```diff
@@ -187,12 +188,15 @@ abstract contract ManagerBase is
         // call into transceiver contracts to send the message
         for (uint256 i = 0; i < numEnabledTransceivers; i++) {
             address transceiverAddr = enabledTransceivers[i];
+
+            bytes32 refundRecipient = refundAddress;
```

Comment:
this is a stack depth issue and re-assignment should be fine here, although i agree a bit ugly. imo block scoping doesn't really make sense here since the `refundAddress` is passed down from the entrypoint (e.g. the lifetime of the variable is really that long).